docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst

```diff
@@ -50,10 +50,11 @@ This sub-subsection specify the attributes of each particular CE of the site. Mu
 | *<CE_NAME>/CEType*                             | Type of CE, can take values as LCG or CREAM                  | CEType = ARC                   |
 +------------------------------------------------+------------------------------------------------------------- +--------------------------------+
 | *<CE_NAME>/LocalCEType*                        | Type of 'Inner' CE, normally empty. Default = "InProcess".   | LocalCEType = Pool             |
-|                                                | Possibilities: potentially all CEs type, but in practice     |                                |
+|                                                | Possibilities: potentially all CE types, but in practice     |                                |
 |                                                | the most valid would be: InProcess, Sudo, Singularity, Pool. |                                |
 |                                                | Pool CE in turn uses InProcess (Default)                     |                                |
 |                                                | or Sudo or Singularity. To specify, use Pool/ce_type.        | LocalCEType = Pool/Singularity |
+|                                                | This option can also go at the Queue level.                  |                                |
 +------------------------------------------------+------------------------------------------------------------- +--------------------------------+
```

Comment:
There is a problem with the table syntax, it is missing a `-` in many of the `----` lines.

Reply:
I have fixed it here.

Reply:
You still have to push this fix, right?